src/app/containers/Navigation/index.jsx

```diff
@@ -30,70 +50,36 @@ const Navigation = () => {
 
   const scrollableListItems = (
     <NavigationUl>
-      {navigation.map((item, index) => {
-        const { title, url } = item;
-        const active = index === 0;
-
-        return (
-          <NavigationLi
-            key={title}
-            url={url}
-            script={script}
-            active={active}
-            currentPageText={currentPage}
-            service={service}
-            dir={dir}
-          >
-            {title}
-          </NavigationLi>
-        );
-      })}
+      {renderListItems(
+        NavigationLi,
+        navigation,
+        script,
+        currentPage,
+        service,
+        dir,
+      )}
     </NavigationUl>
   );
 
-  const dropdownId = 'dropdown-menu';
   const dropdownListItems = (
-    // Hidden attribute is added if we're on amp because the amp toggleVisibility
```

Comment:
This comment was added by Alistair after the swarm review to explain the usage of the `hidden` attribute. I would keep it and put it before the `{cloneElement(dropdownListItems, { id: DROPDOWN_ID, hidden: true })} `.

Reply:
Sure, I'll add a comment back in :)